### PR TITLE
Handle get PBC and active bundle

### PR DESCRIPTION
**Pull request by TerryHowe**
Get a PBC for a specific cluster

* Pass clusterName to the get PBC method
* Do not use kubeadmcontrolplane
* Get active bundle based on PBC for the cluster
* Do not reconcile activeBundle delete on bundle reconciliation

### CODE REVIEW

File: api/v1alpha1/package.go [hunk 1]
```diff
@@ -23,3 +29,24 @@ func (config *Package) GetValues() (values map[string]interface{}, err error) {
 	err = yaml.Unmarshal([]byte(config.Spec.Config), &mapInterfaces)
 	return mapInterfaces, err
 }
+
+func (config *Package) GetClusterName() string {
+	if strings.HasPrefix(config.Namespace, namespacePrefix) {
+		clusterName := strings.TrimPrefix(config.Namespace, namespacePrefix)
+		// Backward compatibility
+		if clusterName == oldPbcName {
+			return os.Getenv(clusterNameEnvVar)
```
Comment (jonahjon) [inline api/v1alpha1/package.go:38]: How exactly does this work?

**Comment by jonahjon** [comment]
Not much code changed just re-arranged. Had one Q but otherwise it looked good.

File: api/v1alpha1/package.go (hunk 1 above)
Comment (TerryHowe) [reply api/v1alpha1/package.go:38]: If the installation has the old PBC name which is something like `eksa-package-bundle-controller`, then use the environment variable to get the cluster name. I probably will follow up with a PR to remove support for this and some migration.

**Pull request merged**